Hw9

pitstt opened the pull request.

Commit:
fix

Commit:
ДЗ
CRUD приложение с Web UI и хранением данных в БД
Цель: Цель: разрабатывать полноценные классические Web-приложения
Результат: Web-приложение полностью на стеке Spring
Необходимо:
1. Создать приложение с хранением сущностей в БД (можно взять библиотеку и DAO/репозитории из прошлых занятий)
2. Использовать классический View на Thymeleaf, classic Controllers.
3. Для книг (главной сущности) на UI должны быть доступные все CRUD операции. CRUD остальных сущностей - по желанию/необходимости.
4. Локализацию делать НЕ нужно - она строго опциональна.

kataus commented on hw09-thymeleaf/src/main/resources/templates/updateAuthor.html:
```diff
@@ -0,0 +1,34 @@
+<form th:action="@{/author}"
+      th:object="${author}" method="POST">
+    Id:
+    <input readonly type="text" th:field="*{id}"/>
```
Настолько update и add формы отличаются чтобы усложнять сопровождение?

pitstt replied:
Удалил add форму

Commit:
ДЗ
CRUD приложение с Web UI и хранением данных в БД
Цель: Цель: разрабатывать полноценные классические Web-приложения
Результат: Web-приложение полностью на стеке Spring
Необходимо:
1. Создать приложение с хранением сущностей в БД (можно взять библиотеку и DAO/репозитории из прошлых занятий)
2. Использовать классический View на Thymeleaf, classic Controllers.
3. Для книг (главной сущности) на UI должны быть доступные все CRUD операции. CRUD остальных сущностей - по желанию/необходимости.
4. Локализацию делать НЕ нужно - она строго опциональна.

kataus commented on hw09-thymeleaf/src/main/java/ru/otus/hw09thymeleaf/service/BookService.java:
```diff
@@ -0,0 +1,42 @@
+
+@Service
+@RequiredArgsConstructor
+public class BookService {
```
А интерфейсы? Как же принцип OpenClose?
